core/src/test/java/org/apache/iceberg/TestPartitionSpecInfo.java

```diff
@@ -20,6 +20,7 @@
 
 import static org.apache.iceberg.types.Types.NestedField.required;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
```

Comment:
@nastra I thought that we specifically did not use static imports. Is it okay for assertions now?

Reply:
static imports for assertions became allowed by this commit
https://github.com/apache/iceberg/commit/b3261d07fc687d5df1f0f8309bc5a0f6022434cf

Reply:
during the JUni4 -> JUnit5 migration we used static imports of `assertThat` for better readability at places that didn't use AssertJ assertions before and switched from JUnit4 asserts to AssertJ asserts. At that point the codebase was split between static import usage of `assertThat` and non-static usage and so we eventually enforced static imports for AssertJ assertions via https://github.com/apache/iceberg/pull/10517